### PR TITLE
[BAEL-4123] What is the purpose of 'Class.forName(“MY_JDBC_DRIVER”)'?

### PR DESCRIPTION
**Pull request by shapirov103**

### CODE REVIEW

File: persistence-modules/core-java-persistence/src/test/java/com/baeldung/jdbc/JdbcDriverLoadingUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            driver = getDriver();
+        }
+        catch (SQLException e) {
+            // expected
+        }
+        assertNull("Must fail on attempt to get a deregistered driver", driver);
+
+        org.postgresql.Driver.register(); // This is invoked as part of Class.forName() logic
+
+        driver  = getDriver();
```
Comment (eric-martin) [inline persistence-modules/core-java-persistence/src/test/java/com/baeldung/jdbc/JdbcDriverLoadingUnitTest.java:61]: Remove extra space before the =

File: persistence-modules/core-java-persistence/src/test/java/com/baeldung/jdbc/JdbcDriverLoadingUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class JdbcDriverLoadingUnitTest {
+
+    @Before
+    public void setup() throws SQLException {
+        if(!org.postgresql.Driver.isRegistered()) {
```
Comment (eric-martin) [inline persistence-modules/core-java-persistence/src/test/java/com/baeldung/jdbc/JdbcDriverLoadingUnitTest.java:16]: Add a space between the if and the (

File: persistence-modules/core-java-persistence/src/test/java/com/baeldung/jdbc/JdbcDriverLoadingUnitTest.java (hunk 1 above)
Comment (shapirov103) [reply persistence-modules/core-java-persistence/src/test/java/com/baeldung/jdbc/JdbcDriverLoadingUnitTest.java]: removed

File: persistence-modules/core-java-persistence/src/test/java/com/baeldung/jdbc/JdbcDriverLoadingUnitTest.java (hunk 2 above)
Comment (shapirov103) [reply persistence-modules/core-java-persistence/src/test/java/com/baeldung/jdbc/JdbcDriverLoadingUnitTest.java]: space added

**Pull request merged**